src/components/graph/graph.renderer.jsx

```diff
@@ -32,8 +32,7 @@ function _buildLinks(nodes, links, linksMatrix, config, linkCallbacks, highlight
         const targetId = target.id || target;
         const key = `${sourceId}${CONST.COORDS_SEPARATOR}${targetId}`;
         const props = buildLinkProps(
-            `${sourceId}`,
-            `${targetId}`,
+            { ...link, source: `${sourceId}`, target: `${targetId}` },
```

Comment:
Here we don't need to pass `source` and `target` anymore, simply pass `link` object without spreading:
```
const props = buildLinkProps(
    link,
    nodes,
    ...
)
```